### PR TITLE
Improve keybind system

### PR DESCRIPTION
**Pull request by PancakeTAS**
Next up command system but I really can't be bothered right now

### CODE REVIEW

**Comment by George417** [comment]
WIP?

File: src/main/java/com/minecrafttas/common/KeybindManager.java
Comment (ScribbleTAS) [inline src/main/java/com/minecrafttas/common/KeybindManager.java:1]: Separate TASmod specific things from this file and extend it in the TASmod package.

File: src/main/java/com/minecrafttas/common/KeybindManager.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	/**
+	 * Register new keybind
+	 * @param keybind Keybind
+	 */
+	public void registerKeybind(Keybind keybind) {
+		this.keybindings.add(keybind);
+		KeyBinding keyBinding = keybind.keyBinding;
+		
+		// block keybinding from virtual keybindings
+		VirtualKeybindings.registerBlockedKeyBinding(keyBinding);
```
Comment (ScribbleTAS) [inline src/main/java/com/minecrafttas/common/KeybindManager.java]: The point with separating blocked keybinds from the normal registering was that in the future I'd maybe add keybinds that can be used by the playback, like dupemod keybinds.

So registering this separately is better imo

**Review by ScribbleTAS — CHANGES_REQUESTED** [review]

File: src/main/java/com/minecrafttas/common/KeybindManager.java (hunk 1 above)
Comment (PancakeTAS) [reply src/main/java/com/minecrafttas/common/KeybindManager.java]: this is the best I can do...

**Review by ScribbleTAS — APPROVED** [review]

**Pull request merged**